languages.yaml

```diff
@@ -79,6 +79,7 @@ Ada:
       - "#0018C9"
       - "#0C0A7C"
     chip: "#02F88C"
+  icon: "\uE6B5" # 
```

Comment:
We should consider removing the comment, as it will only be visible to users who have Nerd Fonts installed and enabled in their code editor.